src/components/dialogs/line-types-catalog/line-types-catalog-selector-dialog.jsx

```diff
@@ -193,14 +193,15 @@ const LineTypesCatalogSelectorDialog = ({ onSelectLine, preselectedRowId, rowDat
         ];
     }, [intl]);
 
-    // Tries to find the preselected row to highlight it.
+    // Tries to find the selected row to highlight it, then scroll to it if needed
     const highlightSelectedRow = useCallback(() => {
         const rowIdToHighlight = selectedRow?.id ?? preselectedRowId;
         if (rowIdToHighlight && rowData) {
             gridRef.current?.api?.forEachNode(function (node) {
                 node.setSelected(node.data?.id === rowIdToHighlight);
             });
         }
+        selectedRow && gridRef.current.api?.ensureNodeVisible(selectedRow);
```

Comment:
```suggestion
        selectedRow && gridRef.current.api?.ensureNodeVisible(selectedRow, 'top');
```

Reply:
Thibault wanted 'middle'. Maybe 'top' is cleaner for the user. But the default value 'bottom' is not handy

Reply:
It asked by PO and more handy

Reply:
OK Done but I had to do more because this part was called everytime that the user selects something, not just when opening the interface.